### PR TITLE
Add linear trend model

### PR DESCRIPTION
**Pull request by guilhermebodin**

### CODE REVIEW

File: src/models.jl [hunk 1]
```diff
@@ -118,5 +118,36 @@ function locallevelmodel(y::VecOrMat{Typ}) where Typ <: AbstractFloat
...
+
+If `y` is proided as an `Array{Typ, 1}` it will be converted to an `Array{Typ, 2}` inside the `StateSpaceModel`.
+"""
+function lineartrendmodel(y::VecOrMat{Typ}) where Typ <: AbstractFloat
+
+    # Number of observations and endogenous variables
+    y = y[:, :]
+    n, p = size(y)
+
+    @info("Creating lenar trend model with $p endogenous variables.")
```
Comment (raphaelsaavedra) [inline src/models.jl]: lenar -> linear

File: test/test_lineartrend.jl [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@testset "Local level model" begin
```
Comment (raphaelsaavedra) [inline test/test_lineartrend.jl]: Local level -> Linear trend

File: src/models.jl (hunk 1 above)
Comment (guilhermebodin) [reply src/models.jl]: ok

File: test/test_lineartrend.jl (hunk 2 above)
Comment (guilhermebodin) [reply test/test_lineartrend.jl]: ok

**Pull request merged**